frontend/src/javascript/stage/Painter.js

```diff
@@ -61,4 +61,28 @@ export class Painter {
     getLayer() {
         return this.layers[this.layerPosition];
     }
-}
+
+    rescalePoints(points) {
+        let minX = Math.min.apply(null, points.map(function(point) { return point.x; }));
+        let maxX = Math.max.apply(null, points.map(function(point) { return point.x; }));
+        let minY = Math.min.apply(null, points.map(function(point) { return point.y; }));
+        let maxY = Math.max.apply(null, points.map(function(point) { return point.y; }));
+
+        let scaleX = 1;
+        let scaleY = 1;
+        let centerX = minX += (maxX - minX) / 2
```

Comment:
IMO trochę za dużo się dzieje w tej linii. Przypisujesz do zmiennej którą w tej samej linii zwiększasz i przypisujesz.

Reply:
Nawet w sumie to wychodzi na to, że centerX będzie miało tą samą wartość co minX. To w sumie takim razie po co nam nowa zmienna. Ja bym pozbył się tego zwiększania minX wogóle.